.nvmrc

```diff
@@ -0,0 +1 @@
+v14.17.3
```

Comment:
Quick question, what impact does this file have for anyone who does not have this version on node installed, given they have nvm anyways?

Reply:
If you have NVM installed, you still need to run `nvm use` to use the version in `.nvmrc`. Unless you have your shell configured to auto-run the command, but that would still happen behind the scenes.

Now, if you don't have the correct version installed, this is the error message you would get:

```
Found '/home/strift/Code/OpenSource/vue-supabase/.nvmrc' with version <v14.17.3>
N/A: version "v14.17.3 -> N/A" is not yet installed.

You need to run "nvm install v4.9.1" to install it before using it.
```